app/assets/javascripts/visualizations/highvis/table.coffee

```diff
@@ -186,11 +194,10 @@ $ ->
             # Restore the search filter type and operator symbol
             operator = $('#' + inputId).closest('tr').find('.soptclass')
             $(operator).attr('soper', column.op)
-            operands = { "eq": "==", "ne": "!", "lt": "<", \
-                         "le": "<=", "gt": ">", "ge": ">=",\
-                         "bw": "^",  "bn": "!^","in": "=", \
-                         "ni": "!=", "ew": "|", "en": "!@",\
-                         "cn": "~",  "nc": "!~","nu": "#", "nn": "!#" }
+            operands = { "eq": "==", "ne": "!",  "lt": "<",  "le": "<=", \
+                         "gt": ">",  "ge": ">=", "bw": "^",  "bn": "!^", \
+                         "in": "=",  "ni": "!=", "ew": "|",  "en": "!@", \
+                         "cn": "~",  "nc": "!~", "nu": "#",  "nn": "!#" }
```

Comment:
ends with - all of these are kind of but not actually in the jqGrid docs. You can see them on iSENSE by looking at the search options for columns